demisto_sdk/commands/pre_commit/tests/test_data/.pre-commit-config_template-test.yaml

```diff
@@ -78,6 +90,10 @@ repos:
       - id: should_be_skipped # only for the unit test
         skip: true
 
+      - id: hook_needs_docker
+        needs:
```

Comment:
whats this?

Reply:
a list of hooks which needed by the hook.
In our use-case, we need `pytest-in-docker` to finish before running `coverage-analyze`